README.md

```diff
@@ -1,8 +1,8 @@
 # Mod2 Week1 Assessment - <YOUR NAME HERE>
 
 ## Setup
-1. Fork this repository
-2. Open your forked repository in Visual Studio
+1. Fork this repository.
+2. Open your forked repository in Visual Studio.
 
 ## Instructions
 1. Insert your name on line 1 to replace `<YOUR NAME HERE>` above.
```

Comment:
It seems obvious to us, but students often make the mistake of not removing the angle brackets. Thus it becomes
`Mod2 Week1 Assessment - <Megan McMahon>`. Do we need to be more explicit about this (not just here, across the board)?